tools/peptideshaker/searchgui.xml

```diff
@@ -168,6 +171,27 @@
                 -myrimatch_max_peak ${myrimatch.myrimatch_max_peak}                    
             #end if
 
+            #*
+            #if $andromeda.andromeda_advanced == "yes"
+                -andromeda_max_pep_mass ${andromeda.andromeda_max_pep_mass}
```

Comment:
@jj-umn here you include the andromeda engine but below it's commented out. I don't think this will work.
Moreover, I don't think we should include Andromeda. The only setup this would be useful is a windwos-only worker and this would mean that all search engines run on windows + SearchGUI. Not sure if this is worth the maintenance and the user confusion on all other installations.
If we could get this bundled with Mono (I made a package for this) this would change the game and make it possible to use this engine on a Linux Cluster as well. What do you think?